deploy-docker-compose.yml

```
@@ -0,0 +1,322 @@
+      - rabbitmq
+      - mongo
+    networks:
+      - app-network
+    restart: always
+
+
+  traefik:
+    image: traefik:v2.9
+    command:
+      - "--api.insecure=true"
+      - "--providers.docker=true" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)